src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs

```diff
@@ -198,5 +200,41 @@ public bool TryGetTable(uint tag, out byte[] table)
         {
             return _typeface.TryGetTableData(tag, out table);
         }
+
+        public bool TryGetStream([NotNullWhen(true)] out Stream? stream)
+        {
+            try
+            {
+                var asset = _typeface.OpenStream();
+                var size = asset.Length;
+                var memoryBase = asset.GetMemoryBase();
+
+                if (memoryBase != IntPtr.Zero)
+                {
+                    unsafe
+                    {
+                        stream = new UnmanagedMemoryStream((byte*)memoryBase, size);
```

Comment:
Who is responsible for releasing memoryBase? Disposing UnmanagedMemoryStream won't do anything IIRC.

Reply:
Skia is responsible for that we could copy the stream to some MemoryStream to avoid this

Reply:
Okay now we always create a copy and the caller is responsible for managing the stream